test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,39 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.LinkedList;
+
+import org.junit.Test;
+
+public class UtilsTest {
+
+    @Test
+    public void null_object_returns_true() {
```

Comment:
Note the naming convention for test methods (the first part is feature/method under test), e.g. `isAnyNull_emptyList_falseReturned`